lib/internal/per_context/primordials.js

```diff
@@ -175,5 +175,17 @@ primordials.SafePromise = makeSafe(
   copyPrototype(original.prototype, primordials, `${name}Prototype`);
 });
 
+// Create copies of abstract intrinsic objects that are not directly exposed
+// on the global object.
+// Refs: https://tc39.es/ecma262/#sec-%typedarray%-intrinsic-object
+[
+  { name: 'TypedArray', original: Reflect.getPrototypeOf(Uint8Array) },
+].forEach(({ name, original }) => {
+  // The static %TypedArray% methods require a valid `this`, but can't be bound,
+  // as they need a subclass constructor as the receiver:
+  copyPrototype(original, primordials, name);
```

Comment:
Isn't that the same as `Promise` above?
```suggestion
  copyPropsRenamedBound(original, primordials, name);
```

Reply:
No, because that would create methods that are bound to the abstract `%TypeError%` constructor, which throws unconditionally.

These methods require a concrete subclass constructor as their `this` parameter.